front/app/containers/ProjectsShowPage/shared/pb/PBExpenses.tsx

```diff
@@ -1,24 +1,15 @@
-import React, { PureComponent } from 'react';
-import { adopt } from 'react-adopt';
+import React, { useState } from 'react';
 import { isNilOrError } from 'utils/helperUtils';
 import { round } from 'lodash-es';
 import moment from 'moment';
+import Tippy from '@tippyjs/react';
```

Comment:
Happy to discuss the (non-)value of this, but if we group the imports in categories as we're doing today this one should be under `//components`